sunkit_pyvista/tests/test_plotting.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import pfsspy
+import pytest
+import pyvista
+from pfsspy import tracing
+from pfsspy.sample_data import get_gong_map
+
+import astropy.constants as const
+import astropy.units as u
+import sunpy.data.test as test
+import sunpy.map as smap
+from astropy.coordinates import SkyCoord
+from sunpy.coordinates import frames
+
+from sunkit_pyvista import SunpyPlotter
+
+pyvista.OFF_SCREEN = True
+
+
+@pytest.fixture
+def aia171_test_map():
+    return smap.Map(test.get_test_filepath('aia_171_level1.fits'))
+
+
+@pytest.fixture
+def plotter():
+    return SunpyPlotter()
+
+
+def test_plot_map(aia171_test_map, plotter, verify_cache_image):
+    plotter.plot_map(aia171_test_map, clip_interval=(0, 99)*u.percent)
+    plotter.plot_solar_axis()
+    plotter.show(cpos=(0, 1, 0), before_close_callback=verify_cache_image)
+
+
+def test_quadrangle(aia171_test_map, plotter, verify_cache_image):
+    bottom_left = SkyCoord(30*u.deg, -10*u.deg,
+                           frame=frames.HeliographicStonyhurst,
+                           obstime=aia171_test_map.date)
+    plotter.plot_quadrangle(bottom_left=bottom_left, width=20*u.deg,
+                            height=60*u.deg, color='blue')
+    plotter.show(cpos=(0, 1, 0), before_close_callback=verify_cache_image)
+
+
+def test_solar_axis(plotter, verify_cache_image):
```

Comment:
Do we not want a solar image as a background for the axis?

Reply:
Same idea as above, I thought we'd test each mesh separately